tests/test_outdated.py

```diff
@@ -41,6 +41,8 @@ class Package:  # noqa: D101
 allowed_outdated: set[str] = {
     "python",  # 3.12.3 needs libexpat >2.6 but VTK not happy about it
     "sphinx",  # 7.3 compat in progress
+    "graphviz",  # conflicts with VTK 9.2.6 via libexpat
+    "mne-rsa",  # 0.91 appeared on conda-forge and was yanked
```

Comment:
that shouldn't be necessary 

i marked it as broken on conda-forge 🤔

Reply:
Must be the `latest-version` we pull from the JSON isn't correct. Feel free to look into it more if you want

Reply:
Yes let me take a look before merging :)

Reply:
Should now work via [50fe246](https://github.com/mne-tools/mne-installers/pull/266/commits/50fe2469cd97d579975cf80f4916e08a114572cf)